### PR TITLE
[EMCAL-524, EMCAL-526] New histograms + fixes for input handling in downscaled modes

### PR DESCRIPTION
**Pull request by mfasDa**
- New histograms requested in EMCAL-524 for the DigitsQC and EMCAL-526 for the raw QC
- Iteration over input container not steered via the the channel type because origin and descriptor
  change in case of downsampling, but instead using the binding which is the same for direct and downsampled channels.

### CODE REVIEW

**Comment by mfasDa** [comment]
Supersedes #797

**Comment by mfasDa** [comment]
The 3 commits are supposed to enter as separate commits though they are in the same PR. The PR must not be squash-merged.

**Comment by mfasDa** [comment]
CI error on mac unrelated (autotools) - builds fine under macOS (Big Sur)

File: Modules/EMCAL/src/RawTask.cxx [hunk 1]
```diff
@@ -378,8 +378,14 @@ void RawTask::monitorData(o2::framework::ProcessingContext& ctx)
...
-  std::vector<framework::InputSpec> filter{ { "filter", framework::ConcreteDataTypeMatcher(dataOrigin, "RAWDATA") } };
-  for (const auto& rawData : framework::InputRecordWalker(ctx.inputs(), filter)) {
+  std::cout << "Found " << ctx.inputs().size() << " inputs" << std::endl;
+
+  auto posReadout = ctx.inputs().getPos("readout");
+  auto nslots = ctx.inputs().getNofParts(posReadout);
+  for (decltype(nslots) islot = 0; islot < nslots; islot++) {
+    auto rawData = ctx.inputs().getByPos(posReadout, islot);
+    auto datahead = framework::DataRefUtils::getHeader<header::DataHeader*>(rawData);
+    std::cout << "Next input " << rawData.spec->binding << "" << datahead->dataOrigin.str << "/" << datahead->dataDescription.str << ", subspec " << datahead->subSpecification << std::endl;
```
Comment (knopers8) [inline Modules/EMCAL/src/RawTask.cxx:388]: That might not go to infologger, if you need it.

**Review by knopers8 — APPROVED** [review]

**Comment by knopers8** [comment]
trusting Markus about the mac build

File: Modules/EMCAL/src/RawTask.cxx (hunk 1 above)
Comment (mfasDa) [reply Modules/EMCAL/src/RawTask.cxx:388]: This is a leftover debug message - should be removed.

**Pull request merged**